### ISSUE TITLE
 Set up super.so for dao.ubq.fi and blog.ubq.fi

### ISSUE THREAD

**Issue by sergfeldman** [ubiquity/uad-ui#266]
- Use the pricing plan Upgraded site https://super.so/pricing

**Comment by nshmadhani** [comment]
This seems like a pretty straight forward bounty to convert all docs/blog pages to notion pages, I would like to start working on it.

**Comment by 0x4007** [comment]
> This seems like a pretty straight forward bounty to convert all docs/blog pages to notion pages, I would like to start working on it.

Unfortunately its not. I just tried the trial of super using dao.ubq.fi and the stylesheet is incompatible so the styles would probably have to be totally redone - yikes. Not sure what the best move here is. I'm tempted to try again with Travis's code but I feel like it isnt a very sustainable solution. 

The alternative is to bite the bullet and redo all of the styles.

**Comment by nshmadhani** [comment]
I would not mind doing the stylesheets again. we could probably try a release in stages and effectively rollout after two weeks.

**Comment by sergfeldman** [comment]
@nshmadhani Please clarify, 
1) Are you willing to complete this bounty for 300 USDC but it will take longer than 2 weeks?
2) What stages of release do you see?

**Comment by 0x4007** [comment]
> we could probably try a release in stages and effectively rollout after two weeks.

I think this is a great idea, but when it comes to design it's really difficult to define concrete deliverables and price points. Anyways with a little bit of trust we can proceed and figure out deliverables/compensation milestones as we work together.

The final objective is to make the super.so instance look exactly like [dao.ubq.fi](https://dao.ubq.fi/) (which is currently hosted with another super.so competitor that is lacking in support and is unstable)

**Comment by nshmadhani** [comment]
@sergfeldman 

1. Are you willing to complete this bounty for 300 USDC but it will take longer than 2 weeks?
    - yes that is what I am getting at, since as @pavlovcik said we need the UI to look exactly like the current one it can take a bit longer to convert the stylesheets.

2. What stages of release do you see?
   - My plan is to first release a basic version with super.so in the coming days and do both the websites seperately.
   - Once we have nailed the stylesheets and are comfortable with one of the websites we can do the next one.

Finanly, if the bounty takes more time I would like the price to be open to negotiation.

**Comment by sergfeldman** [comment]
@nshmadhani You are assigned on this bounty.
Please, post updates after each stage of the release.

**Comment by nshmadhani** [comment]
Where can i find code for the uad blog? I need the notion public page url for creating a super.so website.

**Comment by nshmadhani** [comment]
I got it for the DAO: https://www.notion.so/Ubiquity-DAO-ff1a3cae900941e49cc4d4458cc2867d

**Comment by sergfeldman** [comment]
> 

@pavlovcik @littleflute23 Please, provide the info on the blog.ubq.fi

**Comment by 0x4007** [comment]
Honestly would prefer to consolidate all of the content (dao, blog) onto the dao notion instance. Let's just focus on the dao instance and ignore the blog side for now.

> > 
> 
> @pavlovcik @littleflute23 Please, provide the info on the blog.ubq.fi

**Comment by nshmadhani** [comment]
# Update


TLDR: Trying to replicate the same theme we have at dao.ubq.fi is a bit tedious and the end result wont turn out to be as good as the original.

I have been working on using super.so for dao.ubq.fi. Currently, I have faced many issues while working on it solely due the  fact that injecting code directly into super.so is very restricted.

1. **HTML Code Injection**: super.so does not support adding buttons(HTML) on to the website directly. any html code added will be technically appended and styling it can leave to very inconsistent results across browsers and devices. Due to this adding a switch button for light/dark mode is difficult. However, I can have one consistent theme across the platform. 

2. **Video Background**: In our current implementation we can control where the Notion page is being rendered. Because of this we can control where the background is and add html for the video to load on it. super.so is turning out to be tedious for this.

3.**Pages as Grid**: Each Block for a page is rendered as is on Notion and is not part of special container whose css can be changed for a grid. Current implementation achieves this using `inline-table` but when i tried it on super.so it did not work.

If you have any suggestions as to what I can do please let me know. I have contacted the super.so team on discord/forums and awaiting a response.

Also, are we willing to change the theme for the dao.ubq.fi? should dao.ubq.fi and blog.ubq.fi have the same theme/layout?

I understand that not much work has been done in a week and I have nothing to show for it. At this point, I would understand if you would like to reassign the bounty to someone else.

**Comment by sergfeldman** [comment]
@nshmadhani Thank you for the detailed update! 
Yes, we want dao.ubq.fi and blog.ubq.fi to have the same theme.

We are ready to continue this bounty with you.

**Comment by 0x4007** [comment]
I don't have too much time at the moment to comment but I just wanted to mention that the buttons are pure CSS styling. We can do a lot with CSS!

**Comment by nshmadhani** [comment]
# Update

After working on this bounty for more than 15 days, I will release this bounty back to the pool and to ensure another developer can try to take on this challenge. 

I will be more than willing to answer any questions they might have.

**Comment by sergfeldman** [comment]
@nshmadhani thank you for the update
